src/Microsoft.AspNet.Authentication/DataHandler/TextEncoder.cs

```diff
@@ -2,18 +2,17 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using Microsoft.Framework.Internal;
 
 namespace Microsoft.AspNet.Authentication
```

Comment:
We can if you want, but we don't have anything else in an Internal namespace right now, we would probably move more things there.  Its called by OAuth and OpenId/Twitter today, so its actually getting used.  I was going to make these just private statics before I saw that